src/scripts/language-detection.js

```
@@ -0,0 +1,83 @@
+}
+
+// Get language from cookie
+function getLanguageCookie() {
+  const cookieValue = getCookie("preferred_lang");
+  return cookieValue && locales.includes(cookieValue) ? cookieValue : null;
+}
+
+// Get browser language that matches our supported locales
+function getBrowserLanguage() {
+  // Try to match navigator.languages array first (more specific)
+  if (navigator.languages && navigator.languages.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)